run_bokchoy_tests.sh

```diff
@@ -0,0 +1,29 @@
+i#!/usr/bin/env bash
+
+# this stops the django servers
+stopServers() {
+    kill $(ps aux | grep "[m]anage.py" | awk '{print $2}')
+}
+
+echo "Stop any already running servers..."
+stopServers
+
+echo "Starting Notifications Server..."
+./manage.py runserver --settings=testserver.settings --noreload &
+
+
```

Comment:
maybe we could use 'curl' here to wait for server to respond before proceeding?!?